dependencies.yaml

```
@@ -59,6 +59,10 @@ dependencies:
               cuda: "11.8"
             packages:
               - cudatoolkit=11.8
```

Can we change all these entries like the example below? This is needed to constrain the CUDA version of the RAPIDS packages used for Python testing. (It may be possible to derive the correct constraint from a pin on `cudatoolkit` but that's going to be less direct for the environment solver.)
```suggestion
              - cuda-version=11.8
              - cudatoolkit
```